docs/.gitignore

```diff
@@ -0,0 +1,4 @@
+/index.adoc
+/index.html
```

Comment:
Would it be useful to have the documentation as part of the repo? Or should it be regenerated in each build? I guess that latter would be better, correct?

Reply:
No, it would not include it in the repo, because when starting, the docs are also available on the server via swagger.